index.html

```diff
@@ -10,28 +10,239 @@
-        <li class="block">
-            <img class="img-container" src="./img/project_images/communicate.png" alt="image:communicate">
-            <p class="block-info">Communicate directly</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/book.png" alt="image:book-online">
-            <p class="block-info">Book online</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/review.png" alt="image:guests-reviews">
-            <p class="block-info">Real guest reviews</p>
-        </li>
-    </ul>
-</section>
+<main>
+    <section class="main">
+        <div class="main-container">
+            <div class="main-header">
+                <img src="img/project_images/main-logo-icon.png" alt="" class="main-header-logo">
+                <ul class="main-header-list">
+                    <li class="main-header-items">Stays</li>
+                    <li class="main-header-items">Attractions</li>
+                    <li class="main-header-items night-icon"></li>
+                    <li class="main-header-items account-icon"></li>
+                </ul>
+            </div>
+            <h1 class="main-title">Discover stays<br>
+                to live, work or just relax</h1>
+            <div class="main-search">
+                <form action="/" method="POST" class="search-form">
+                    <label for="destination" class="label-search"><span class="label">Your destination or hotel
+                        name</span><br>
+                        <input id="destination" class="search-input input-place" type="text" name="place"
+                               placeholder="New York"></label>
+
+                    <label for="date" class="label-search"><span class="label">Check-in - Check-out</span><br>
+                        <input id="date" class="search-input input-date" type="text" name="date"
+                               placeholder="Tue 15 Sept—Sat 19 Sept"></label>
+
+                    <label for="conditions">
+                        <input id="conditions" class="search-input input-conditions" type="text" name="stay-conditions"
+                               placeholder="2 Adults—0 Children—1 Room">
+                    </label>
+                    <button type="button" class="button button-search-blue">Search</button>
+                </form>
+            </div>
+            <ul class="main-applications">
+                <li class="main-applications-google">
+                    <a href="#" class="main-applications-link">
+                        <svg class="main-applications-icon">
+                            <use xlink:href="#google-play"></use>
+                        </svg>
+                    </a>
+                </li>
+                <li class="main-applications-apple">
+                    <a href="#" class="main-applications-link">
+                        <svg class="main-applications-icon">
```

Comment:
So cool that you add svg-sprite 👑 👍 👏 
hovewer, it's better to keep svg-sprite in its own file